tests/test_user_profile.py

```diff
@@ -91,3 +95,30 @@ def test_read_profile_for_player(self):
             datetime(2021, 2, 12, 0, 15, 0): 0.5,
             datetime(2021, 2, 12, 0, 30, 0): 0.1
         })
+
+    def test_correct_time_expansion_read_arbitrary_profile(self):
```

Comment:
Following our style convention i would propose:
```suggestion
    def test_read_arbitrary_profile_returns_pofile_with_correct_time_expansion(self):
```

Reply:
Forgot to leave a comment here, should be fixed by now !